src/System.Drawing.Common/src/System/Drawing/Font.cs

```diff
@@ -77,6 +77,8 @@ public sealed partial class Font : MarshalByRefObject, ICloneable, IDisposable,
         /// Gets the face name of this <see cref='Font'/> .
         /// </summary>
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
+        [Editor("System.Drawing.Design.FontNameEditor, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", "System.Drawing.Design.UITypeEditor, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+        [TypeConverter(typeof(FontConverter.FontNameConverter))]
```

Comment:
This needs to be a string as well.  See https://github.com/dotnet/corefx/pull/42263/files#diff-8d8bf4bc5bf97b89dbbd44547df284d0R18